### PR TITLE
{lib}[foss/2020b] ucx-py v0.21.0 w/ Python/3.8.6

### PR DESCRIPTION
**Pull request by zdemat**
Python module of ucx-py called ucp requires numpy and pynvml so the build is in foss.
Documentation page refers to itself as UCX-Py, however the post on pypi.org: https://pypi.org/project/ucx-py/ is lowercase ucx-py.

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/u/ucx-py/ucx-py-setup-get_ucx_incpath.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+--- setup.py
```
Comment (smoors) [inline easybuild/easyconfigs/u/ucx-py/ucx-py-setup-get_ucx_incpath.patch:1]: @zdemat please add author + short description to the top of the patch?

Comment (zdemat) [reply easybuild/easyconfigs/u/ucx-py/ucx-py-setup-get_ucx_incpath.patch:5]: Patch description and author added.

**Comment by smoors** [comment]
@zdemat thanks for doing the updates.
I've made some additional changes, adding the version to the patch name.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by zdemat — COMMENTED** [review]
@smoors the comment in checksums is then clearly wrong, not sure if easybuild has issues with that or if CI can find that. It is still somehow understandable for humans.

**Comment by smoors** [comment]
@zdemat yes, missed that, fixed.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node309.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/2432a7707cdc1376e82e3bd23c129628 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @zdemat!

**Pull request merged**